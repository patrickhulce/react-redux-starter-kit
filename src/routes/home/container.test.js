import React from 'react'
import renderer from 'react-test-renderer'

import Home from './container'

describe('routes/home/container.js', () => {
  it('should render', () => {
    const mockChild = <div className="child" />
    const component = renderer.create(<Home>{mockChild}</Home>)

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
