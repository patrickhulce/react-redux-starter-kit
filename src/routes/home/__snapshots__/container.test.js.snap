exports[`routes/home/container.js should render 1`] = `
<div
  className="container">
  <h1>
    <i
      className="fa fa-home" />
     Home
  </h1>
  <a
    onClick={[Function]}
    style={Object {}}>
    Contact
  </a>
  <div
    className="child" />
</div>
`;
